core/vm/operations_verkle.go

```diff
@@ -23,16 +23,16 @@ import (
 )
 
 func gasSStore4762(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize uint64) (uint64, error) {
-	return evm.Accesses.TouchSlotAndChargeGas(contract.Address().Bytes(), common.Hash(stack.peek().Bytes32()), true, contract.Gas, true), nil
+	return evm.Accesses.TouchSlotAndChargeGas(contract.Address().Bytes(), common.Hash(stack.peek().Bytes32()), true, true /* XXX needs a missing API */, contract.Gas, true), nil
```

Comment:
this is going to overcharge everything, but we don't have a function to check if a storage existed. A possibility would be to check if the previous value was 0, which would approximate it better while still being incorrect. Gotta sleep on it.